suites/apis/googleServiceAccountRemovalTest.js

```diff
@@ -0,0 +1,471 @@
+  // test that the clean up job does not remove access to valid SA/projects
+
+  // Setup
+  const googleProject = fence.props.googleProjectDynamic;
+  await fence.complete.forceLinkGoogleAcct(users.user0, googleProject.owner);
+
+  // Register account
+  let registerRes = await fence.do.registerGoogleServiceAccount(
+    users.user0,
+    googleProject,
+    ['QA']
+  );
+  fence.ask.responsesEqual(registerRes, fence.props.resRegisterServiceAccountSuccess);
+
+  // Lock the project
+  lockRes = await google.lockGoogleProject(googleProject);
+  chai.expect(lockRes, 'Could not lock project').to.be.true;
+
+  // Get creds to access data
+  let [pathToKeyFile, keyFullName] = await google.createServiceAccountKeyFile(googleProject);
+
+  // Access data
+  user0AccessQARes = await google.getFileFromBucket(
+    fence.props.googleBucketInfo.QA.googleProjectId,
+    pathToKeyFile,
+    fence.props.googleBucketInfo.QA.bucketId,
+    fence.props.googleBucketInfo.QA.fileName
+  );
+
+  // Delete the creds so that the clean up job will not mark this SA as invalid
+  await google.deleteServiceAccountKey(keyFullName);
+  files.deleteFile(pathToKeyFile);
+
+  // Run clean up job
+  checkAndCleanSA();
+
+  // Get creds to access data
+  [pathToKeyFile, keyFullName] = await google.createServiceAccountKeyFile(googleProject);
+
+  // Access data
+  user0AccessQAResAfter = await google.getFileFromBucket(
+    fence.props.googleBucketInfo.QA.googleProjectId,
+    pathToKeyFile,
+    fence.props.googleBucketInfo.QA.bucketId,
+    fence.props.googleBucketInfo.QA.fileName
+  );
+
+  // Clean up
+  console.log('cleaning up');
+
+  await google.deleteServiceAccountKey(keyFullName);
+  files.deleteFile(pathToKeyFile);
+
+  await fence.do.deleteGoogleServiceAccount(
+    users.user0,
+    googleProject.serviceAccountEmail,
+  );
+
+  await fence.do.unlinkGoogleAcct(
+    users.user0,
+  );
+
+  // Unlock the project
+  let unlockRes = await google.unlockGoogleProject(googleProject);
+
+  // Asserts
+  chai.expect(user0AccessQARes,
+    'User should have bucket access before clean up job'
+  ).to.have.property('id');
+
+  chai.expect(user0AccessQAResAfter,
+    'User should have bucket access after clean up job'
+  ).to.have.property('id');
+
+  chai.expect(unlockRes, 'Could not unlock project').to.be.true;
+}).retry(2);
+
+
+// TODO: enable. this test fails until the jenkins envs are setup with the new monitoring SA
+xScenario('SA removal job test: monitor SA does not have access @reqGoogle', async (fence, users, google, files) => {
+  // test invalid SA because the monitor does not have access
+
+  // Setup
+  const googleProject = fence.props.googleProjectDynamic;
+  let getMonitorRes = await fence.do.getGoogleServiceAccountMonitor(users.user0);
+  fence.ask.assertStatusCode(getMonitorRes, 200, 'Could not get SA monitor info');
+  let monitorEmail = getMonitorRes.body.service_account_email;
```

Comment:
this could fail if the monitoring endpoint changes what it returns (i.e. no `service_account_email` field). should make sure the tests respond with a sensical error in that case